pages/glossary/matching-characters.md

```diff
@@ -5,4 +5,4 @@ unambiguous: true
 objective: true
 ---
 
-A sequence of characters is considered to _match_ another if, after removing leading and trailing [space characters](https://html.spec.whatwg.org/#white_space) and replacing remaining occurrences of one or more space characters with a single space, the two sequences of characters are equal character-by-character, ignoring any differences in letter casing.
+A sequence of characters is considered to _match_ another if, after removing leading and trailing [whitespace characters][] and replacing remaining occurrences of one or more whitespace characters with a single space, the two sequences of characters are equal character-by-character, ignoring any differences in letter casing.
```

Comment:
```suggestion
A sequence of characters is considered to _match_ another if, after removing leading and trailing [whitespace characters][] and replacing remaining occurrences of one or more whitespace characters with a single space, the two sequences of characters are equal character-by-character, ignoring any differences in letter casing.

[whitespace characters]: #whitespace 'Definition of Whitespace'
```
I'm actually surprised that we don't have a test to catch these missing links in definitions 🤔

Reply:
#2146, in the meantime 🤷‍♀️